### PR TITLE
[shopsys] accessories is provided by elasticsearch for ListedProductView

### LINKED ISSUE #2102: Provide accessories from elasticsearch
### What is happening
In ListedProductViewElasticFacade we have methods for obtaining accessories (`getAccessories()`, `getAllAccessories()`), that loads them from database. We could update those methods as in #2090 was accessories added into elasticsearch.
### Expected result
Accessories is provided by from elasticsearch in `ListedProductViewElasticFacade`.

### PR DESCRIPTION
**Pull request by s3tezsky**
| Q             | A
| ------------- | ---
|Description, reason for the PR| In elasticsearch implementation there is accessories provided from PostgreSQL. This PR is changing it.
|New feature| Yes 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| fixes #2102  
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: packages/framework/src/Model/Product/Search/FilterQueryFactory.php [hunk 1]
```diff
@@ -238,10 +238,15 @@ public function createVisibleProductsByProductIdsFilter(array $productIds): Filt
...
      * @return \Shopsys\FrameworkBundle\Model\Product\Search\FilterQuery
      */
-    public function createSellableProductsByProductIdsFilter(array $productIds): FilterQuery
+    public function createSellableProductsByProductIdsFilter(array $productIds, ?int $limit = null): FilterQuery
     {
-        return $this->createVisibleProductsByProductIdsFilter($productIds)->filterOnlySellable();
+        $filterQuery = $this->createVisibleProductsByProductIdsFilter($productIds)->filterOnlySellable();
+        if ($limit === null) {
+            return $filterQuery;
+        }
```
Comment (TomasLudvik) [inline packages/framework/src/Model/Product/Search/FilterQueryFactory.php:249]: Please separate this part of code from others by new lines

**Review by TomasLudvik — CHANGES_REQUESTED** [review]
Hello @s3tezsky, nice job, mostly with catching that error in ordering. I have one think to ask you - can you please move that fix of ordering to separate commit, so if somebody is looking why did you make this change, he can read it commit message? It was difficult for me at beginning to catch why you did that change. Thank you.

File: packages/framework/src/Model/Product/Search/FilterQueryFactory.php (hunk 1 above)
Comment (s3tezsky) [reply packages/framework/src/Model/Product/Search/FilterQueryFactory.php:249]: I do not see any need to add 2 more blank lines. This is pretty straightforwaqrd as is.

**Comment by s3tezsky** [comment]
> can you please move that fix of ordering to separate commit

I cannot for 2 reasons.
1. It is not pretty clear how it should be splitted. Besides the commit is not too large and it is pretty obvious.
2. Doing so will cost me some amount of time which I do not want to waste on that.

File: packages/framework/src/Model/Product/Search/FilterQueryFactory.php (hunk 1 above)
Comment (s3tezsky) [reply packages/framework/src/Model/Product/Search/FilterQueryFactory.php:253]: Ok, I have added it 😉

**Review by TomasLudvik — APPROVED** [review]

**Review by grossmannmartin — COMMENTED** [review]
does it close #2102?

**Review by sslt — APPROVED** [review]

**Comment by s3tezsky** [comment]
> does it close #2102?

It does, thanks!

**Pull request merged**